traits/stubs_tests/util.py

```diff
@@ -165,5 +165,7 @@ def assertRaisesMypyError(self, filepath):
         for line, error_codes in parsed_mypy_output.items():
             if line not in line_error_map or sorted(
                     line_error_map[line]) != sorted(list(error_codes)):
-                s = "\n{}\n{}".format(str(filepath), normal_report)
+
+                s = "Unexpected error on line {}\n{}\n{}".format(
```

Comment:
Drive-by improvement to report the line number that caused a test to fail. (Otherwise it's hard to understand what went wrong from the test output.)